### PR TITLE
Refactor Saying Service

### PR DESCRIPTION
**Pull request by NimperX**
Refactor Saying Service #88

### CODE REVIEW

**Comment by CrazyMarvin** [comment]
Can you please have a look, @markuscodes1?

File: package.json [hunk 1]
```diff
@@ -1,58 +1,58 @@
 {
```
Comment (markuscodes1) [inline package.json:1]: Please remove this changes. Only the format has been updated

File: src/app/services/saying.service.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ export class SayingService {
     this.httpClient
       .get<any>('https://ipinfo.io/json')
       .subscribe(e => {
```
Comment (markuscodes1) [inline src/app/services/saying.service.ts:54]: Please use a meaningful variable name

File: src/app/services/saying.service.ts [hunk 3]
```diff
@@ -52,7 +52,7 @@ export class SayingService {
     this.httpClient
       .get<any>('https://ipinfo.io/json')
```
Comment (markuscodes1) [inline src/app/services/saying.service.ts:53]: Please create a dto/type instead of using any

**Review by markuscodes1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (NimperX) [reply package.json:1]: I didn't catch you well @markuscodes1. Can you please clarify this change

Comment (markuscodes1) [reply package.json:1]: Sure. I think it's not necessary to merge the `package.json` in this PR. There are no functional changes. Only the format has been updated. So I think it's out of scope for this pr.

**Pull request merged**